topologies/kne/init_arista.config

```diff
@@ -0,0 +1,39 @@
+transceiver qsfp default-mode 4x10G
```

Comment:
Can we merge this with arista_ceos.config?

https://github.com/openconfig/featureprofiles/blob/main/topologies/kne/arista_ceos.config

I think we can rename the result to just "arista.config"

@bstoll what do you think?

Reply:
I was able to successfully run the test with arista_ceos.config so I will replace init_arista.config with this

Reply:
Is this file still needed in the PR?

Reply:
The only difference is this line "username admin privilege 15 role network-admin nopassword" which I believe allows ssh connection without password. It's ok, I can remove the file.